vuln-test.py

```
@@ -23,7 +23,7 @@
 app = Flask(__name__)
 
 # 脆弱: SECRET_KEY をソースに直書き
-app.config['SECRET_KEY'] = 'supersecretkey123'
+app.config['SECRET_KEY'] = 'secretkey123456789'
```

Ruff (0.12.2)

Possible hardcoded password assigned to: "SECRET_KEY"

(S105)